models/app.models.js

```diff
@@ -13,5 +14,12 @@ exports.fetchEndpoints = () => {
     .then((endpoints) => {
         return endpoints
     })
+}
 
+exports.fetchArticleById = (article_id) => {
+    
+    return database.query(`SELECT * FROM articles WHERE article_id = $1`, [article_id])
+    .then((article) => {
+        return article.rows
+    })
```

Comment:
You can make a small change here to ensure that the object is being returned and not the full ```rows``` array